addons/website/views/website_templates.xml

```diff
@@ -402,7 +402,7 @@
         </li>
         <li groups="website.group_website_publisher" class="list-inline-item">
             <t t-set="url_return" t-value="url_for('', '[lang]') + '?' + keep_query()"/>
-            <a t-attf-href="/web#action=base.action_view_base_language_install&amp;website_id=#{website.id if website else ''}&amp;url_return=#{url_return}">
```

Comment:
Forget about this too: https://drive.google.com/file/d/1BwdRHLCa-e9ouF3LVJ3cAMYH2kVap1JE/view, seems not thought through (and what you did was not what was exactly asked anyway).

Reply:
Fred asked me to do this but anyway I removed it :smile:

Reply:
I removed `t-if="len(languages) &gt; 1"` but what about this https://drive.google.com/file/d/1KS5r8iAZSnKtk9-7tagw_4hzjPK8awZx/view , should be done or also removed ? ( `d-none d-md-block d-print-block` )

Reply:
You can handle the mobile stuff. I would not put d-print-block though